constants/index.js

```diff
@@ -0,0 +1,17 @@
+const RESPONSE_MSG = {
+    SUCCESS: "Request Successful.",
+    FAILED: "Something is not good.",
+}
+const RESPONSE_STATUS = {
+    SUCCESS: 0,
+    FAILED: -1
+}
+const STATUS = {
+    SUCCESS: 0,
+    FAILED: -1
+}
+
+const ARRAYLENGTH = 0
+const SAMESCHEME = 1.00.toFixed(2)
```

Comment:
why use toFixed() function here
never use any function in constant
let constant be constant